src/App.jsx

```diff
@@ -1,9 +1,131 @@
+            </section>
+
+            <section>
+                <h2>Best verkochte tv</h2>
+                <article className="best-selling-container">
+                  <span>
+                    <img src={bestSellingTv.sourceImg} alt="Afbeelding van het product"/>
+                  </span>
+                    <div className="best-selling-product-information">
+                        <h3 className="tv-name">{tvName(bestSellingTv)}</h3>
+                        <p className="tv-price">{tvPrice(bestSellingTv.price)}</p>
+                        <p>{tvScreenSize(bestSellingTv.availableSizes)}</p>
+                        <ul className="tv-assets">
+                            <li><img src={check} alt="Icoon: check" className="icon"/>wifi</li>
+                            <li><img src={minus} alt="Icoon: minus" className="icon"/>speech</li>
+                            <li><img src={check} alt="Icoon: check" className="icon"/>hdr</li>
+                            <li><img src={check} alt="Icoon: check" className="icon"/>bluetooth</li>
+                            <li><img src={minus} alt="Icoon: minus" className="icon"/>ambilight</li>
+                        </ul>
+                    </div>
+                </article>
+            </section>
+
+            <section className="brands-container">
+                <h2>Beschikbare merken</h2>
+                <ul className="product-brands">
+                    {inventory.map((tv) => {
+                        // eslint-disable-next-line react/jsx-key
+                        return <li key={tv.type}>{tv.brand}</li>
+                        // hoe zorg je dat dubbele merken niet voorkomen?
+                    })}
+                </ul>
+            </section>
+
+            <section>
+                <h2>Alle tvs</h2>
+                <button type="button" onClick={sortBestSelling}>Meest verkocht eerst</button>
+                <button type="button" onClick={sortCheapest}>Goedkoopste eerst</button>
+                <button type="button" onClick={sortSport}>Meest geschikt voor sport eerst</button>
+                <button type="button" onClick={sortLargest}>Grootste schermgroottes eerst</button>
+
+                {inventory.map((tv) => {
+                    return (
+                        <article className="products-container" key={tv.type}>
+                            <span className="product-image">
+                                <img src={tv.sourceImg} alt="Afbeelding van het product"/>
+                            </span>
+                            <div className="product-information">
+                            <h3>{tvName(tv)}</h3>
+                            <p className="all-tv-price">{tvPrice(tv.price)}</p>
+                            <p>{tvScreenSize(tv.availableSizes)}</p>
+                            <ul className="tv-assets">
+                                {tv.options.map((option) => {
+                                    if (option.applicable === true) {
```

Comment:
Dit kun je nog verkorten:
```suggestion
                                    if (option.applicable) {
```